### PR TITLE
[M552] Update bleaching card

### PR DESCRIPTION
**Pull request by tienhoah**
Screenshot 2023-09-14 at 1 05 16 PM

### CODE REVIEW

**Comment by tienhoah** [comment]
[Update the bleaching data vis card on the dashboard.](https://trello.com/c/nEReywLU/552-update-the-bleaching-data-vis-card-on-the-dashboard)

File: src/components/CardChartContent.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { histogramContentPropType, pieChartContentPropType } from '../lib/mermai
 
 const CardChartContent = ({
   chartType,
-  protocolName,
+  sampleUnit,
```
Comment (julia-rom) [inline src/components/CardChartContent.js:12]: nice clearer naming

File: src/components/InformationCard.js [hunk 2]
```diff
@@ -50,8 +50,8 @@ const InformationCard = ({
   histogramContent,
   isFiltering,
   isPrivatePolicy,
-  protocol,
-  protocolName,
+  protocolProperties,
```
Comment (julia-rom) [inline src/components/InformationCard.js:53]: maybe use `sampleUnitProperties` for consistency if we are switching `protocolName` to` sampleUnit`

File: src/components/InformationCard.js [hunk 3]
```diff
@@ -126,8 +126,12 @@ InformationCard.propTypes = {
   isPrivatePolicy: PropTypes.bool,
   pieChartContent: pieChartContentPropType,
   projectFishFamilies: PropTypes.arrayOf(PropTypes.string),
-  protocol: PropTypes.oneOfType([fishbeltPropType, benthicPitPropType, bleachingPropType]),
-  protocolName: PropTypes.string,
+  protocolProperties: PropTypes.oneOfType([
```
Comment (julia-rom) [inline src/components/InformationCard.js:129]: same here, suggestion to use `sampleUnitProperties`

**Review by julia-rom — APPROVED** [review]
looks good, would prefer to see consistent naming (`sampleUnit` everywhere instead of `protocol`)

**Pull request merged**